manifests/rhoai/params.env

```diff
@@ -1 +1 @@
-odh-training-operator-controller-image=docker.io/kubeflow/training-operator:v1-855e096
+odh-training-operator-controller-image=quay.io/opendatahub/training-operator:latest
```

Comment:
I think that's preferable to avoid using a rolling tag. That can be a SHA at the moment if there is no existing fixed tag.

Reply:
thank you, I've updated the tag